rustls/src/crypto/mod.rs

```diff
@@ -588,6 +566,46 @@ pub fn default_fips_provider() -> CryptoProvider {
     aws_lc_rs::default_provider()
 }
 
+mod crypto_default_provider {
+    use alloc::sync::Arc;
+
+    #[cfg(not(feature = "std"))]
+    use alloc::boxed::Box;
+
+    #[cfg(not(feature = "std"))]
+    use once_cell::race::OnceBox;
+    #[cfg(feature = "std")]
+    use once_cell::sync::OnceCell;
+
+    use crate::crypto::CryptoProvider;
+
+    #[cfg(feature = "std")]
+    static PROCESS_DEFAULT_PROVIDER: OnceCell<Arc<CryptoProvider>> = OnceCell::new();
+    #[cfg(not(feature = "std"))]
+    static PROCESS_DEFAULT_PROVIDER: OnceBox<Arc<CryptoProvider>> = OnceBox::new();
+
+    #[cfg(feature = "std")]
+    pub(crate) fn install_default_provider(
```

Comment:
For these method names, I'd prefer just using the method names (`install_default()`, `get_default()`). Suffixing with `_provider` isn't all that helpful since they're in a `crypto_default_provider` module already.

Reply:
shortened ✅